sections/06-vibrant-bodies.md

```diff
@@ -0,0 +1,201 @@
+
+While _In Passing_'s "system to see with" is outwardly transformational, capable of changing a viewer's sense of time and perceptual energies, it also operates its own internal mode of transformation at a technical level. The image processing algorithm that causes the leaf to slowly erode is driven by two time-varying values: the threshold, which specifies the degree to which the leaf should be consumed by its background, and the noise seed, a value that influences the random sparkle pattern and its rate of change. These parameters are generated by a digital synthesizer, a software component ordinarily intended to generate sounds, which has been repurposed to modulate the parameters of the visual transformation of the leaf. In the second section of _In Passing_, the crackling sound is produced by connecting this synthesizer directly to the speakers instead of to the visual effect. Here, the synthesizer acts as a flexible "translation mediator," to use Don Ihde’s terminology (112), mapping the sonic to the visual and back again.
+
+![Signal graph for In Passing](../images/in-passing-signal-graph.png)
+
+_Figure x: Diagram illustrating the signal path of oscillators and modulators in In Passing's synthesis algorithm._
+
+Yet there is something troubling to me about Ihde's idea of translation, at least when thinking about how _In Passing_ operates. The idea of translation is predicated upon an informational model of the digital (an admittedly very dominant model at the heart of the system, as illustrated by the definitional term "information technology"), rather than one of the energetics, signals, and flows described earlier. Ihde's translational mediation is based on an abstraction of data from its representation. "To become available the information, data, or image must be transformed, translated, into what is open to our anthropological constant, an embodied human" (113). The power of Ihde's model is that it positions embodiment centrally within the realm of the digital, but it also carries with it a sedimented transmission model of communication, positioning data "above" representation as a (relatively) stable referent or message preserved within a linguistic system of translation.
+
+Indeed, _In Passing_’s repurposable synthesizer algorithm can be interpreted informationally as proof of data's translatability across representations--we successfully both see and hear its influence. However, Bruno Latour offers an alternative concept of transformation, one that rejects the idea of data as raw material. He argues that there is no such thing as information, "there is only transformation. Information as something which will be carried through space and time, without deformation, is a complete myth" (Lovink and Shultz).
+
```

Comment:
Comment from David: "It’s a minor thing but I think important: the paragraph break after the Lovink and Schultz note is a bit misleading and potential obscures your argument. Also, the statement 'In Passing’s repurposable synthesizer algorithm can be interpreted informationally as proof of data's translatability across representations--we successfully both see and hear its influence' should be reworded: your whole point is that that would be an incorrect interpretation, so don’t say that it can be interpreted that way! Instead, say something like 'one might mistakenly read the work as…'